_includes/_companies.html

```diff
@@ -76,6 +76,30 @@
       </div>
 
       <div>
+         <a href="http://oxx.co.id">
+            <image src="images/logo_oxx.png">
+         </a>
+         <h3>Oxx</h3>
+         <p>Teknologi untuk kebutuhkan bisnis dan organisasi.</p>
+      </div>
+
+      <div>
+         <a href="http://taxies.co.id">
```

Comment:
this domain is expired, it needs to be excluded :) or else reactivate the domain